.github/workflows/update-release-draft.yml

```diff
@@ -10,20 +10,20 @@ on:
 jobs:
   update_release_draft:
     runs-on: ubuntu-latest
-    if: github.repository == 'jellyfin/jellyfin-plugin-webhook'
+    if: github.repository == 'jellyfin/jellyfin-plugin-omdb'
```

Comment:
```suggestion
    if: github.repository == 'jellyfin/jellyfin-plugin-webhook'
```

Reply:
![You didn't see anythjing](https://media.giphy.com/media/C87IXdLfJ44Zq/giphy.gif)